extensions/amp-story-interactive/0.1/test/test-amp-story-interactive-slider.js

```diff
@@ -78,5 +78,31 @@ describes.realWin(
         ampStorySlider.getRootElement().querySelector('input[type="range"]')
       ).to.not.be.null;
     });
+
+    it('should be disabled after the input event', async () => {
+      await ampStorySlider.buildCallback();
+      await ampStorySlider.layoutCallback();
+      const slider = ampStorySlider
+        .getRootElement()
+        .querySelector('input[type="range"]');
+      // simulates a change event, which is when the user releases the slider
+      slider.dispatchEvent(new CustomEvent('change'));
+      expect(slider.hasAttribute('disabled')).to.be.true;
+    });
+
+    it('checks that input is the same as the number displayed in the bubble', async () => {
+      await ampStorySlider.buildCallback();
+      await ampStorySlider.layoutCallback();
+      const slider = ampStorySlider
+        .getRootElement()
+        .querySelector('input[type="range"]');
+      const sliderValue = ampStorySlider
```

Comment:
`sliderBubbleSpan` might be a good name for this, since it's getting an element.
"Value" feels like it might refer to a number.